modules/core/src/lib/attribute/attribute-transition-utils.ts

```diff
@@ -156,16 +156,16 @@ export function padBuffer({
   }
 
   const toData = isConstant
-    ? attribute.value
-    : attribute.getBuffer().getData({srcByteOffset: byteOffset});
+    ? (attribute.value as TypedArray)
+    : (attribute.getBuffer() as Buffer).getData();
   if (attribute.settings.normalized && !isConstant) {
     const getter = getData;
     getData = (value, chunk) => attribute.normalizeConstant(getter(value, chunk));
   }
 
   const getMissingData = isConstant
     ? (i, chunk) => getData(toData, chunk)
-    : (i, chunk) => getData(toData.subarray(i, i + size), chunk);
+    : (i, chunk) => getData(toData.subarray(i + byteOffset, i + byteOffset + size), chunk);
```

Comment:
This is not typing, but a bug fix?

Reply:
`Buffer.getData` in v9 no longer accept arguments, and this was not caught because strict check was off. So yes I guess it's a bug fix.